attributes/default.rb

```diff
@@ -18,3 +18,4 @@
 #
 
 default['build-essential']['compile_time'] = false
+default['build-essential']['msys']['path'] = 'C:/msys'
```

Comment:
So following the pattern we use for other Windows cookbooks like `wix` and `7-zip` I would make this:

```ruby
default['build-essential']['msys']['path'] = "#{ENV['SYSTEMDRIVE']}\\msys"
```
or
```ruby
default['build-essential']['msys']['path'] = File.join(ENV['SYSTEMDRIVE'], 'msys')
```